django/contrib/postgres/forms/jsonb.py

```diff
@@ -0,0 +1,29 @@
+import json
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+__all__ = ['JSONField']
+
+
+class JSONField(forms.CharField):
+    default_error_messages = {
+        'invalid': _("'%(value)s' value must be valid JSON."),
+    }
+
+    def __init__(self, **kwargs):
+        kwargs.setdefault('widget', forms.Textarea)
+        super(JSONField, self).__init__(**kwargs)
+
+    def to_python(self, value):
+        try:
+            return json.loads(value)
+        except ValueError:
```

Comment:
> If this is a genuine security concern

I don't know that it is; it just triggered my spidey-sense. Again in theory this should be OK, since JSON is designed to be safe in this way (unlike, say, YAML). I'm doing some digging right now, but having a look at the Ruby vulns I mentioned, they're usually related to "extra things" that various folks added on top, not the core JSON functionality. 

I don't think this should block merge, but it is something I want to investigate further.